### ISSUE TITLE
DB list freezes for long time when there are many (>1k) DBs

### ISSUE THREAD

**Issue by SAPikachu** [ermouth/couch-photon#30]
It would be great if the list can be lazy-loaded, or alternatively provide an option to disable loading of DB information in such cases (to avoid freezing the UI and pressuring the server). Thank you!

**Comment by ermouth** [comment]
That option already created very long ago: click the rightmost button at the tabs panel, and you may discover `DB list cols` setting, which allows disabling both loading security descriptors and sizes.

However, I had a point in backlog to add more comprehensive behavior.

**Comment by ermouth** [comment]
Btw, what would you say if favorites sublist always loads security and size info regardless of that settings?

**Comment by SAPikachu** [comment]
> That option already created very long ago: click the rightmost button at the tabs panel, and you may discover DB list cols setting, which allows disabling both loading security descriptors and sizes.
Oh I was not aware of that, it works for me, thank you!

> Btw, what would you say if favorites sublist always loads security and size info regardless of that settings?
In my case it doesn't really matter, but I see that it would be a nice addition for many cases. :)

**Comment by ermouth** [comment]
Investigating another complaint of freezing DB list I discovered the reason might be non-native scroll turned on. With native scroll the issue is nearly negligent.

So if you are on Mac which supports `overflow:overlay` CSS rule, consider turning native scroll on. Non-native scroll plugin was added to emulate `overflow:overlay` on PCs. 

To turn native scroll on go to settings (rightmost button at navbar), and set Scroll sensitivity to Native, like this: <img width="442" alt="Снимок экрана 2020-12-09 в 21 03 31" src="https://user-images.githubusercontent.com/1540047/101669155-9ff87b80-3a62-11eb-8fe2-3fdab3f98a9e.png">
